Library/Homebrew/cask/cask_loader.rb

```diff
@@ -236,7 +236,9 @@ def self.for(ref)
       when 2..Float::INFINITY
         loaders = possible_tap_casks.map(&FromTapPathLoader.method(:new))
 
-        raise TapCaskAmbiguityError.new(ref, loaders)
+        raise TapCaskAmbiguityError.new(ref, loaders) if loaders.map(&:tap).map(&:name).size == 1
```

Comment:
Can you explain the `loaders.map(&:tap).map(&:name).size` bit?

Reply:
Sorry, I would like to write `loaders.map(&:tap).map(&:name).uniq.size` . I'll fix it soon.